modules/space/rendering/renderableorbitalkepler.h

```diff
@@ -35,6 +35,63 @@
 #include <ghoul/misc/objectmanager.h>
 #include <ghoul/opengl/programobject.h>
 
+namespace {
```

Comment:
The reason was to make the PropertyInfo available to the satellites and smallbody subclasses, in order to prevent redundant info in each subclass. Agreed it's not consistent so will change this. I was talking with Alex today and he brought up a better way to do it.